.github/workflows/ingest-cadet-metadata.yml

```diff
@@ -79,6 +79,13 @@ jobs:
           DATAHUB_TELEMETRY_ENABLED: false
         run: poetry run datahub ingest -c ingestion/cadet.yaml
 
+      - name: associate datasets from cadet with their database
+        env:
+          DATAHUB_GMS_TOKEN: ${{ secrets.DATAHUB_GMS_TOKEN }}
+          DATAHUB_GMS_URL: ${{ vars.DATAHUB_GMS_URL }}
+          DATAHUB_TELEMETRY_ENABLED: false
+        run: poetry run datahub ingest -c ingestion/assign_cadet_databases.yaml
```

Comment:
how come this is a source rather than a transformer on the DBT source? (I was expecting this to work the same as the assign_cadet_domains one)

Reply:
On the one hand, it creates its own separate aspects, but on the other hand, it is intended to modify the exact set of entities touched by the DBT source, so in my mind it seems like part of the model ingestion 🤷🏻

Reply:
I initially has it as another transformer, but I changed it into another source as part of me trying to get it to work. AFAIK it should work as another transformer as well.